pmd-java/src/main/java/net/sourceforge/pmd/cpd/JavaTokenizer.java

```diff
@@ -62,7 +62,7 @@ protected TokenEntry processToken(Tokens tokenEntries, GenericToken currentToken
 
         if (ignoreLiterals && (javaToken.kind == JavaTokenKinds.STRING_LITERAL
                 || javaToken.kind == JavaTokenKinds.CHARACTER_LITERAL
-                || javaToken.kind == JavaTokenKinds.DECIMAL_LITERAL
```

Comment:
Note that this is a bug, that should be fixe on master. `DECIMAL_LITERAL` cannot match any token, because it's declared with a `#`.

Reply:
I guess, we should also add TEXT_LITERAL to this list of ignored literals. We can fix both parts with #2159 .